include/openenclave/internal/syscall/sys/stat.h

```diff
@@ -45,9 +46,24 @@ OE_EXTERNC_BEGIN
 #define OE_S_IRWGRP (OE_S_IRGRP | OE_S_IWGRP)
 #define OE_S_IRWOTH (OE_S_IROTH | OE_S_IWOTH)
 
-#define __OE_STAT oe_stat
-#include <openenclave/internal/syscall/sys/bits/stat.h>
-#undef __OE_STAT
+OE_STATIC_ASSERT((sizeof(struct oe_stat_t) % 8) == 0);
```

Comment:
Python libclang bindings can help with auto-generating these asserts:
https://eli.thegreenplace.net/2011/07/03/parsing-c-in-python-with-clang